### DISCUSSION TITLE
The engine "node" is incompatible with this module

### DISCUSSION THREAD

**Discussion by Njimukara** [vercel/community#1174] [Help] [answered]
### Summary

Is there a workaround for an error i'm getting on vercel during deployment.
"""The engine "node" is incompatible with this module. Expected version "12 - 16". Got "18.12.1""""

### Additional information

The engine "node" is incompatible with this module. Expected version "12 - 16". Got "18.12.1"

**Comment by amyegan** [comment, marked as answer]
Hi @Njimukara! You seem to have a dependency installed that is not compatible with node version 18. Try changing the [Node.js Version](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/node-js#node.js-version) in your project settings from 18 to 16, then redeploy.

**Reply by Njimukara** [reply]
I have implemented the changes as suggested @amyegan, however, the problem still persists
![image](https://user-images.githubusercontent.com/52873523/208696260-029a2897-cb76-44a3-a7c4-70c6bafcb26e.png)
Here's the node version which I changed
![image](https://user-images.githubusercontent.com/52873523/208696520-53deb5c7-9fe9-40e9-ae9f-c30eb246d84f.png)
And here's the same error message

Thanks in advance for your help :-)

**Reply by amyegan** [reply]
Do you have a node engine set in your `package.json`? If so try removing that setting, or try changing it to 
```
{
  "engines": {
    "node": "16"
  }
}
```

**Reply by Njimukara** [reply]
Thanks a lot @amyegan, I added node version 16 in my package.json and it solved the issue
